src/analyzer/Analyzer.h

```diff
@@ -626,6 +627,9 @@ class Analyzer {
 	 */
 	virtual unsigned int MemoryAllocation() const;
 
+	static void SetIgnoreChecksumsNets(TableValPtr t);
```

Comment:
I know they are rather obvious - but it would be great if we could still add documentation to both calls. We try doing this for everything new that is added.